### PR TITLE
Multiple Sources using collections

### PR DESCRIPTION
**Pull request by geramirez**
@konklone and I paired yesterday to develop this version of including multiple sources. Still needs some front-end work but the structure is here.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -32,4 +46,7 @@ twitter: usgsa
 
 sass:
   sass_dir: sass
-  style: nested
\ No newline at end of file
+  style: nested
+
+url: https://pages.geramirez.gov/analytics.usa.gov
```
Comment (konklone) [inline _config.yml]: Did you mean to commit this `url` to this PR? Also, `pages.geramirez.gov`? :)

File: _config.yml [hunk 2]
```diff
@@ -15,12 +15,26 @@ exclude:
...
+    scope:
+      path: ""
+      type: agencies
+    values:
+      layout: default
+
+collections:
+  agencies:
+    output: true
+    permalink: /:path/
```
Comment (konklone) [inline _config.yml]: In our previous work, we used `/agencies/:path/`. I'm up for either one, I just want to make sure this is deliberate.

**Comment by konklone** [comment]
@geramirez @gbinal @tdlowden What do you see as necessary to get this into an MVP state for deploying?

Right now, this is how it looks:

image

File: _config.yml (hunk 2 above)
Comment (geramirez) [inline _config.yml]: Yep, I figured using `/:agency` pattern would be easier to remember either way works for me.

Comment (konklone) [inline _config.yml]: `/:agency` works just fine, and is easier to remember -- let's stick with that.

**Comment by gbinal** [comment]
Let's discuss at today's meeting but I'm a :+1: for soft-launching any agencies that have given us a green light and we can separately decide what the MVP is for sharing with others.

**Pull request merged**